src/elements/content-sidebar/additional-tabs/AdditionalTab.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/variables';
+@import '../../../styles/common/_animations';
+
+.bdl-AdditionalTab-icon,
+.bdl-AdditionalTab-moreIcon {
+    @include bdl-fadeIn(.5s);
+
+    height: 24px;
+    width: 24px;
+}
+
+.bdl-AdditionalTab-moreIcon .fill-color {
+    fill: $downtown-grey;
+}
+
+.bdl-AdditionalTab--is-hidden {
```

Comment:
Modifiers are indicated with a single word, like `--primary`. States are indicated with a separate class, like `.bdl-is-hidden` or just `.is-hidden`.